tony-core/src/main/java/com/linkedin/tony/LocalizableResource.java

```diff
@@ -34,6 +35,8 @@
   private LocalResourceType resourceType;
```

Comment:
What's the difference between `path` and `localFileName`? And how do they differ from `sourceFilePath`? Can you add some Javadoc explaining the differences?

Reply:
agreed, this is confusing, will do some housekeeping for this class

Reply:
Thanks. The new name `completeResourceString` and the Javadoc make it much clearer.